lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -496,6 +496,39 @@ static int process_cmd(const tunnel_comand *cmd, command_cb cb, void *ctx) {
             return 0;
         }
 
+        case TunnelCommand_RemoveIdentity: {
+            tunnel_delete_identity delete_id = {0};
+            if (cmd->data == NULL || parse_tunnel_delete_identity(&delete_id, cmd->data, strlen(cmd->data)) != 0) {
+                result.success = false;
+                result.error = "invalid command";
+                free_tunnel_delete_identity(&delete_id);
+                break;
+            }
+            struct ziti_instance_s *inst = model_map_get(&instances, delete_id.identifier);
+
+            if (inst == NULL) {
```

Comment:
This series of conditionals feels a bit convoluted. I wouldn't mention it but this pattern is repeated a few times.

First the error string and success status are set if `inst == NULL`; after which the `break` on line 513 takes us out of the `TunnelCommand_RemoveIdentity` case.